lib/awful/tag.lua

```diff
@@ -873,6 +874,14 @@ function tag.mt:__call(...)
     return tag.new(...)
 end
 
+-- Extend the luaobject
+object.capi_index_fallback(capi.tag, {
+    getter_class    = tag             ,
+    setter_class    = tag             ,
+    getter_fallback = tag.getproperty ,
+    setter          = tag.setproperty ,
```

Comment:
IMO, it make the code faster to read as it highlight the structure and code affinity. Just as tabs/space, this is a polarizing topic with no general consensus. I wont go religious on this, but I prefer that style and it would not be the first place where it is in use (but there is not that many). The big problem with that style is "git-blame"/patch noise unless you enable the ignore-whitespace option.